### DISCUSSION TITLE
Merging both services into one file so that they can use the same IP

### DISCUSSION THREAD

**Discussion by shrivastavshubham34** [minio/minio#17872] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Set up a service that can access both the web console and s3 on the same IP. Since Minio supports one cert for both s3 and web

**Describe the solution you'd like**
When setting up the helm chart maybe you can combine both services into one. Just adding a different port for each s3 and web. Kind of like what bitnami does [https://github.com/bitnami/charts/blob/main/bitnami/minio/templates/service.yaml]

**Describe alternatives you've considered**
As of now, I'm using externalIPs to set the same IP for both services. But it still takes an extra IP.

**Additional context**
Current workaround
![image](https://github.com/minio/minio/assets/13406071/60684e0d-8c06-4fb8-8e25-3737318c6ef6)

**Comment by harshavardhana** [comment, marked as answer]
@shrivastavshubham34 good idea, feel free to send a PR.
